sandbox/docs/JavaScript/Delegators/Delegator.Ajax.html.erb

```diff
@@ -102,6 +102,55 @@
 <p><a id="tip" data-behavior="BS.Tooltip" title="The tootlip!">this link has a tip</a></p>
 	<% end %>
 
+
+    <h3>Error handling</h3>
+
+    <p>There are two methods for error handling ajax failures. You can specify a <code>errorRedirectURL</code> which will send the browser to a new page upon failure, or you can specify <code>failureTriggers</code> to invoke.</p>
+
+    The code block is a little hard to read, but here's what that looks like:
+
+    <pre><code>&lt;a class="btn btn-default"
+    data-trigger="ajax Stop"
+    data-ajax-options="
+      {
+        'useSpinner':true,
+        'action':'replace',
+        'target':'self',
+        'failureTriggers':{
+          '!&gt;div::addClass':{
```

Comment:
it's a convention that already exists in other things. `selector::trigger: {config}`